cli/remotebazel/remotebazel.go

```diff
@@ -1037,14 +1039,20 @@ func parseArgs(commandLineArgs []string) (bazelArgs []string, execArgs []string,
 		return nil, nil, status.WrapError(err, "parse remote bazel cli flags")
 	}
 
-	// Ensure all bazel remote runs use the remote cache.
-	// The goal is to keep remote workloads close to our servers, so use the same
-	// app backend as the remote runner.
-	bazelArgs = arg.Remove(bazelArgs, "bes_backend")
-	bazelArgs = arg.Remove(bazelArgs, "remote_cache")
-	bazelArgs = append(bazelArgs, "--config=buildbuddy_bes_backend")
-	bazelArgs = append(bazelArgs, "--config=buildbuddy_bes_results_url")
-	bazelArgs = append(bazelArgs, "--config=buildbuddy_remote_cache")
+	if len(bazelArgs) > 0 {
+		if *script != "" {
+			return nil, nil, fmt.Errorf("cannot pass both a bazel command and --script")
+		}
+
+		// Ensure all bazel remote runs use the remote cache.
+		// The goal is to keep remote workloads close to our servers, so use the same
+		// app backend as the remote runner.
+		bazelArgs = arg.Remove(bazelArgs, "bes_backend")
```

Comment:
Maybe we should add these in the [bazel wrapper in ci_runner:main ](https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/cmd/ci_runner/main.go#L2683) instead of here, to guarantee that these flags are set on every bazel command. With the current implementation, these will no longer be applied if a user uses --script